Fix following error message:

$ /usr/gcc/9/bin/go build main.go
# command-line-arguments
ar: bad option: -D
usage:
    ar -d [-SvV] archive [file...]
    ar -m [-SvV] [-{a|b|i} posname] archive [file...]
    ar -p [-sSvV] archive [file ...]
    ar -q [-cSvV] archive [file...]
    ar -r [-cSuvV] [-{a|b|i} posname] archive [file...]
    ar -t [-sSvV] archive [file...]
    ar -x [-CsSTvV] archive [file...]
    ar [-sSvV] archive

The error is harmless, no functionality issue, GCCGO runs Solaris "ar" which
doesn't know -D option, after failure, GCCGO runs it again without -D.

--- gcc-13.1.0/libgo/go/cmd/go/internal/work/gccgo.go
+++ gcc-13.1.0/libgo/go/cmd/go/internal/work/gccgo.go
@@ -48,7 +48,7 @@ }
 func (gccgoToolchain) ar() string {
 	ar := cfg.Getenv("AR")
 	if ar == "" {
-		ar = "ar"
+		ar = "/usr/gnu/bin/ar"
 	}
 	return ar
 }
--- gcc-13.1.0/libgo/go/go/internal/gccgoimporter/importer_test.go
+++ gcc-13.1.0/libgo/go/go/internal/gccgoimporter/importer_test.go
@@ -180,7 +180,7 @@ }
 
 		runImporterTest(t, imp, initmap, &test)
 
-		cmd = exec.Command("ar", "cr", afile, ofile)
+		cmd = exec.Command("/usr/gnu/bin/ar", "cr", afile, ofile)
 		out, err = cmd.CombinedOutput()
 		if err != nil {
 			t.Logf("%s", out)
